operators/cutters/complex_split_boolean.py

```
@@ -34,7 +34,6 @@ def poll(cls, context):
 
     def draw(self, context):
         layout = self.layout
-        box = layout.box()
         col = box.column()
```

the line
col = box.column()
can be removed as well. otherwise it should throw an error, because "box" is not defined.  did you not test it?